contracts/src/payments/PaymentVault.sol

```diff
@@ -0,0 +1,119 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {OwnableUpgradeable} from "@openzeppelin-upgrades/contracts/access/OwnableUpgradeable.sol";
+import {PaymentVaultStorage} from "./PaymentVaultStorage.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+/**
+ * @title Entrypoint for making reservations and on demand payments for EigenDA.
+ * @author Layr Labs, Inc.
+**/
+contract PaymentVault is PaymentVaultStorage, OwnableUpgradeable {
+ 
+    constructor(
+        uint256 _reservationBinInterval,
+        uint256 _reservationBinStartTimestamp,
```

Comment:
hey arch! Would you please remove this timestamp? I think this won't be needed as we chatted offline:)?